docs/multi-platform.md

```diff
@@ -0,0 +1,109 @@
+# Building multi-platform images
+
+| :zap: For building multi-platform images with `docker buildx`, see [the `docker buildx` documentation](https://github.com/docker/buildx#building-multi-platform-images). |
+|--------------------------------------------------------------------------|
+
+BuildKit provides built-in support for building multi-platform images using QEMU usermode emulators.
+
+```bash
+buildctl build \
+  --frontend dockerfile.v0 \
+  --opt platform=linux/amd64,linux/arm64 \
+  --output type=image,name=docker.io/username/image,push=true \
+  ...
+```
+
+You do not need to set up QEMU manually in most cases.
+
+## Troubleshooting
+
+### Error `exec user process caused: exec format error`
+
+You may face an error like `exec user process caused: exec format error`, mostly when you are using a third-party package of BuildKit that lacks
+`buildkit-qemu-*` binaries.
+
+In such a case, you need to register QEMU to `/proc/sys/fs/binfmt_misc` by one of the following steps:
+
+<details>
+<summary>Debian and Ubuntu</summary>
+<p>
+
+```bash
+sudo apt-get install -y qemu-user-static
+```
+
+Make sure to install `qemu-user-static` package, not `qemu-user-binfmt` package.
+
+</p>
+</details>
+
+<details>
+<summary>Fedora</summary>
+<p>
+
+```bash
+sudo dnf install -y qemu-user-static
+```
+
+Make sure to install `qemu-user-static` package, not `qemu-user-binfmt` package.
+</p>
+</details>
+
+<details>
+<summary>RHEL and its derivatives</summary>
+<p>
+Unlike Fedora, the `qemu-user-static` package is not available for RHEL (as of RHEL 8.4) and its derivatives such as Rocky Linux.
+
+Use `tonistiigi/binfmt` container image instead.
+
+</p>
+</details>
+
+<details>
+<summary>openSUSE</summary>
+<p>
+
+openSUSE provides `qemu-linux-user` package, however, this package is not compatible with BuildKit (fails with an error `exec user process caused: no such file or directory`
+due to an unresolvable symlink).
+
+So, use `tonistiigi/binfmt` container image instead.
+
+Also, you might need to use `kernel-default` kernel instead of `kernel-default-base` kernel if you face an error related to `binfmt_misc`.
+
+</p>
+
+</details>
+
+
+<details>
+<summary>Arch Linux</summary>
+<p>
+
+```bash
+sudo modprobe binfmt_misc
+yay -Syu qemu-user-static-bin binfmt-qemu-static-all-arch
+```
+
+[`qemu-user-static-bin`](https://aur.archlinux.org/packages/qemu-user-static-bin/) and
+[`binfmt-qemu-static-all-arch`](https://aur.archlinux.org/packages/binfmt-qemu-static-all-arch/)
+are AUR pacakges.
+
+</p>
+
+</details>
+
+<details>
+<summary>Docker image (`tonistiigi/binfmt`)</summary>
```

Comment:
```suggestion
<summary>Docker image (<code>tonistiigi/binfmt</code>)</summary>
```

Looks like it's not rendered properly:

![image](https://user-images.githubusercontent.com/1951866/136183869-71b2f24d-a4b2-43cb-bd78-c112f0354882.png)